src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs

```diff
@@ -536,6 +534,8 @@ private void TestBasicNavigation(string appName, bool usesAuth = false, bool ski
                 Browser.Equal(appName.Trim(), () => Browser.Title.Trim());
             }
 
+            // Skipping to workaround https://github.com/dotnet/runtime/issues/38154
+            skipFetchData = true;
```

Comment:
Also, please either revert this (if it's fixed now) or add to the list of things we're tracking